composer.json

```diff
@@ -19,14 +29,21 @@
         "dealerdirect/phpcodesniffer-composer-installer": "^0.4",
         "sirbrillig/phpcs-variable-analysis": "^2.0",
         "wimg/php-compatibility": "^8.0",
-        "wp-coding-standards/wpcs": "^0.14.0"
+        "wp-coding-standards/wpcs": "^0.14.0",
+        "phpunit/phpunit": "^6.2",
+        "antecedent/patchwork": "2.0.8",
+        "brain/monkey": "^2.0",
+        "mockery/mockery": "~0.9",
+        "wpfulcrum/extender": "^3.0"
     },
     "config": {
         "sort-order": true
     },
     "scripts": {
         "install-codestandards": [
             "Dealerdirect\\Composer\\Plugin\\Installers\\PHPCodeSniffer\\Plugin::run"
-        ]
+        ],
+        "cs-check": "vendor/bin/phpcs --colors",
```

Comment:
This colors arg can be defined in the phpcs.xml.dist. See https://github.com/squizlabs/PHP_CodeSniffer/blob/9b6260b4ff9410682fd34a34e68e377e60dc4961/phpcs.xml.dist#L13